# Request
nav_msgs/OccupancyGrid map
---
# response to frontiers
roadmap_builder/Frontier[] frontiers
